src/Main.java

```diff
@@ -20,7 +20,7 @@ public static void main(String[] args) throws IOException {
                 break;
             case 1: {
                 if (args[0].equals("gui")) {
-                    new Gui();
+                    new Gui(new ObjectMapper());
```

Comment:
Ezt emeld ki egy közös változóba, ugyanis 2 alkalommal is kell ez, valamint az erre készült metódust hívd meg, hogy a GUI is tudjon kétféle JSON-t olvasni.